File: soknad/src/main/java/no/nav/syfo/kafka/soknad/dto/SoknadPeriodeDTO.java
package no.nav.syfo.kafka.soknad.dto;

import lombok.Builder;
import lombok.Value;

import java.time.LocalDate;

@Value
@Builder(toBuilder = true)
public class SoknadPeriodeDTO {
    LocalDate fom;
    LocalDate tom;
    Integer grad;
}

File: soknad/src/main/kotlin/no/nav/syfo/kafka/soknad/dto/SoknadPeriodeDTO.kt
package no.nav.syfo.kafka.soknad.dto

import java.time.LocalDate

data class SoknadPeriodeDTO(
        val fom: LocalDate? = null,
        val tom: LocalDate? = null,
        val grad: Int? = null,
        val antallBehandlingsdagerUke: Int? = null
)
